ministryofjustice/laa-apply-for-criminal-legal-aid
# Add self employed exit

## Description of change
Employed journey is only open to applicants who are employed ONLY, not self-employed or employed AND self-employed.

- Adds a new eforms exit page for self-employed
- Conditionally displays content on eforms exit page if employed_journey feature flag is on or not
- Remove conditionality on means_journey feature flag (we can add it back in if for some reason means gets switched back off)

## Notes for reviewer
Copy has been checked with content design

## Screenshots of changes (if applicable)

### Before changes:
<img width="1068" alt="image" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/45827968/e0993633-6f35-469b-be64-17b301b99a1c">

### After changes:
IF employed journey is enabled AND any combination of self-employed is selected
<img width="1031" alt="image" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/45827968/6972ae39-ffa1-4ac2-be49-69ab867e4d26">

## How to manually test the feature
1. Get to income journey page 1 and select Employed only, expect to see existing eForms exit page ("they are not working")
2. Get to income journey page 1 and select Self-employed only, expect to new eForms exit page
3. Get to income journey page 1 and select Self-employed AND employed, expect to new eForms exit page
